packages/vx-brush/src/BaseBrush.tsx

```diff
@@ -33,6 +33,7 @@ export type BaseBrushProps = {
   onClick?: (event: MouseHandlerEvent) => void;
   clickSensitivity: number;
   disableDraggingSelection: boolean;
+  resetOnEnd: boolean;
```

Comment:
one nit – can we make this optional? `resetOnEnd?`

Reply:
Yep absolutely, should have thought of that 😅